### ISSUE TITLE
Problematic clear input button 

### ISSUE THREAD

**Issue by meganvilla** [OpenSpace/OpenSpace#3119]
When a user starts entering a search query, a clear input button is displayed. The  button is implemented as an icon with role="button" and a tabindex attribute to make it  focusable but keyboard-only users cannot press the Enter or Space to activate the  button. Additionally, a title attribute with the value of "Clear input field" is relied upon to  provide context for the control but this is not surfaced to many groups of users. 
Note that the `<i>` tag holds the uninformative text of "cancel" which will make the  purpose of the button unclear to screen reader users (See: Incorrect Implementation of  Icons).

Suggested Solution 
Implement the control as a native `<button>` element and use the aria-label attribute to  provide it with an accessible name, e.g. aria-label="Clear input field" or aria label="Clear search entry". 

Severity: 1
